src/cozy/wrappers/installation-page/installation-page.component.ts

```diff
@@ -14,6 +14,7 @@ interface InstallationPageProps {
     bitwardenData: {
         extension_installed: boolean;
     };
+    onSkipExtension: any;

```

Comment:
Is it really `any`? `function` or something?

Reply:
Forgot to refactor this part. I'll do it, thanks

Reply:
I added a simple function type : https://github.com/cozy/cozy-pass-web/pull/9/commits/6af0d98954d5d956f1c5b1108796b5e50e813319